index.d.ts

```diff
@@ -37,7 +37,7 @@ export interface RenderCounterProps {
   start?: Function;
   pauseResume?: Function;
   reset?: Function;
-  update?: (newEnd?: number) => void;
+  update?: UpdateFunction;
```

Comment:
Same as aboves feedback of all other pauseResume, start, update, reset functions.